### DISCUSSION TITLE
texture orentation density

### DISCUSSION THREAD

**Discussion by lyg1232** [mtex-toolbox/mtex#1385] [Ask Anything] [answered]
Hello!
I measured the texture of some magnesium-lithium alloys with xrd, and drew the pole figure and odf diagram with mtex,  I want to get the texture orientation density of the
![1](https://user-images.githubusercontent.com/81472625/158762923-02b1b595-46c1-478e-aa9f-d8f991565f37.png)
 texture on this red line in the odf diagram, as shown in Figure c.But I don't know how to get it, Thank you for your help.

**Comment by kilir** [comment, marked as answer]
Hi,
how about this:
```
mtexdata dubna
odf = calcODF(pf)
o = orientation.byEuler([0:360]*degree,35*degree,15*degree,odf.CS)
plot(o.phi1/degree,eval(odf,o))
```
Cheers,
Rüdiger

**Reply by lyg1232** [reply]
Thanks，it works
